### ISSUE TITLE
lingering steps for water and soil example data

### ISSUE THREAD

**Issue by turbomam** [microbiomedata/submission-schema#42]
- checkout `main` and run `make test`
  - clean up with `git stash` and or `make schema_cleanup clean`
- copy contents of console to a text editor like Sublime Text
- sort lines
- make unique
- copy any errors or warnings below
- there may also be actionable "INFOs". Would require additional investigation.
---
Also look for commented lines in the example data files

Remember, ideally the invalid examples would only have one invalid aspect, so they could be fixed and then would pass.

Are we going to keep all of these?  What are they good for?
- example data files in `examples/`
- `src/data/examples`... see `tests/test_data.py`
- `src/data/invalid` and `src/data/valid`... see `examples/output/README.md` target in `project.Makefile`

**Comment by turbomam** [comment]
This "error" is expected and desired

```shell
! poetry run check-jsonschema \
    --schemafile project/jsonschema/nmdc_submission_schema.schema.json \
    src/data/invalid/SampleData-water-data-alkalinity-list.yaml
```
>Schema validation errors were encountered.
  src/data/invalid/SampleData-water-data-alkalinity-list.yaml::$.water_data[0].alkalinity: ['50 milligram per liter', '100 spoons per bucket'] is not of type 'string'

**Comment by turbomam** [comment]
>WARNING:root:No namespace defined for URI: https://microbiomedata/schema/ammonium_nitrogen
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/ecosystem
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/ecosystem_category
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/ecosystem_subtype
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/ecosystem_type
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/lbc_thirty
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/lbceq
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/manganese
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/nitrate_nitrogen
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/nitrite_nitrogen
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/specific_ecosystem
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/zinc
WARNING:root:Not implemented: slot
WARNING:root:There is no established path to nmdc_submission_schema - compile_python may or may not work

**Comment by turbomam** [comment]
Examine `local/slot_usage.tsv`

**Comment by turbomam** [comment]
```
  RelToOxygenEnum:
    permissible_values:
      aerobe:
      anaerobe:
      facultative:
      microaerophilic:
      microanaerobe:
      obligate aerobe:
      obligate anaerobe:
```

---

```
  rel_to_oxygen_enum:
    permissible_values:
      aerobe:
      anaerobe:
      facultative:
      microaerophilic:
      microanaerobe:
      obligate aerobe:
      obligate anaerobe:
```

---

```
  oxy_stat_samp_enum:
    permissible_values:
      aerobic:
      anaerobic:
      other:
```

**Comment by turbomam** [comment]
These just need to be all-lower-cased! Thanks @pkalita-lbl 

```
#    micro_biomass_C_meth: xxx
#    micro_biomass_N_meth: xxx
#    microbial_biomass_C: 111 units
#    microbial_biomass_N: 111 units
```

**Comment by turbomam** [comment]
multi-valued titles:
- analysis/data type
- history/tillage

How are MIxS triad and GOLD pull-downs getting created?

Which are really MIxS, MIxS modified etc.

What help is displayed

Time and date formats

Time zone assumption/validation/labeling

**Comment by mslarae13** [comment]
@turbomam is the outstanding task here to just update example files?

**Comment by pkalita-lbl** [comment]
@turbomam Can you summarize the outstanding actionable tasks for this issue? Or can this issue be closed?

**Comment by turbomam** [comment]
Closing — the specific items here are from early 2023 and the schema and example data have changed substantially since. Any outstanding example data issues should be filed as new, specific issues. Thanks @pkalita-lbl for the nudge.
